Usersignuppage.php

```diff
@@ -0,0 +1,91 @@
+<?php
+  include 'partials/nav.php';
+?>
+
+<?php
+
+if(isset($_POST['register'])){
+  include 'partials/db_connect.php';
+  $email = $_POST['email'];
+  $pass = $_POST['password'];
+  $cpass = $_POST['confirm_password'];
+  $existsql = "SELECT * FROM `user_details` WHERE email='$email'";
+  $result = mysqli_query($link, $existsql);
+  $numExistrows = mysqli_num_rows($result);
+
+  if($numExistrows > 0){
+    echo '
+    <div class="alert alert-secondary alert-dismissible fade show" role="alert" Style="margin-top:5px;">
+      <strong> Password do not match or username already Exist!"</strong>
+      <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+    </div>
+    ';
+  } else {
+    if($pass == $cpass){
+      $hash = password_hash($pass, PASSWORD_DEFAULT);
+      $sql = "INSERT INTO `user_details` (`email`, `password`) VALUES ('$email', '$hash')";
+      $result = mysqli_query($link, $sql);
+      if($result){
+        echo '
```

Comment:
You cannot echo a whole HTML code, Thats not a good practice. Find alternative ways to fix this.